Automattic/jetpack
# Add response inbox app and endpoint

Still WIP

## Proposed changes:
- Move REST endpoint out of the plugin and into the package, testing with v4
- Add form responses inbox app and store

### Other information:


## Jetpack product discussion
pcsBup-Pl-p2

## Does this pull request change what data or activity we track or use

## Testing instructions:
Enable new package and dashboard, add these 2 lines at the top of `projects/plugins/jetpack/modules/contact-form.php`:
```php
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

Run `jetpack build packages/forms && jetpack install packages/forms && jetpack install plugins/jetpack`.

Head to `wp-admin/admin.php?page=jetpack-forms`, you should see the new form responses inbox. This PR is merely functional and even without much polishing (styles are still skewed) being able to see the inbox and your form responses should be enough. This is a WIP towards a fully fledged inbox app for form responses.
